assets/javascripts/lib/discourse-markdown/kolide-recheck.js

```diff
@@ -0,0 +1,44 @@
+import { parseBBCodeTag } from "pretty-text/engines/discourse-markdown/bbcode-block";
+import I18n from "I18n";
+
+function addRecheckButton(buffer, matches, state) {
+  const parsed = parseBBCodeTag(matches[0], 0, matches[0].length);
+
+  if (!parsed.attrs.issue) {
+    return;
+  }
+
+  let token = new state.Token("a_open", "a", 0);
+  token.attrs = [
+    ["class", "kolide-recheck"],
+    ["href", "#"],
+    ["data-issue", parsed.attrs.issue],
+  ];
+  buffer.push(token);
+
+  token = new state.Token("text", "", 0);
+  token.content = I18n.t("discourse_kolide.button.recheck");
+  buffer.push(token);
+
+  token = new state.Token("a_close", "a", -1);
+  buffer.push(token);
+
+  return;
```

Comment:
This does not do anything.

Reply:
I'm using the hyperlink (`<a>`) tag as a button. If I didn't return false, the page will navigate to the top (`href="#"`).